packages/phrases/src/locales/zh-cn.ts

```diff
@@ -269,12 +247,8 @@ const translation = {
       created_password: '初始密码：',
       created_button_close: '关闭',
       created_button_copy: '复制',
```

Comment:
These two can be put in `general` as well

Reply:
updated